### ISSUE TITLE
feature to exclude certain **scanners** from being involved in any distance calculations, so that (say) a vacuum with a scanner mounted on it won't mess up the calculations, but that's a different thing from what you are looking to solve.

### ISSUE THREAD

**Issue by Dtrieb** [agittins/bermuda#314]
Oh, sidenote.. there will be a feature to exclude certain **scanners** from being involved in any distance calculations, so that (say) a vacuum with a scanner mounted on it won't mess up the calculations, but that's a different thing from what you are looking to solve.

_Originally posted by @agittins in https://github.com/agittins/bermuda/issues/295#issuecomment-2305076956_

Is the feature to exclude scanners online?

**Comment by agittins** [comment]
No, it's not planned to be present until trilateration is implemented in #21 

Is there a use-case where this is causing an issue for you? Bear in mind the current beta v0.6.9rc2 includes per-proxy calibration so you can apply an offset that will address a certain proxy being more sensitive than others.

**Comment by Dtrieb** [comment]
The Problem is that my RPi itself is stronger than all other devices and so I am always in the room were my RPi is.

**Comment by agittins** [comment]
Cool, that's definitely a thing that the per-proxy calibration should fix. Can you give v0.6.9rc2 a try, and in "Configure", "Calibration 2: Per-scanner RSSI Offsets", adjust the offset on your rpi (probably "hci0") so that it doesn't overwhelm the other measurements?

You need to select a device to calibrate against (eg, your phone) then click "Submit", then it will give you distances measured by each proxy. Put in a negative offset on your rpi and "submit" to update the measurements, repeat until you find an offset that works nicely, tick the "Save and close" box and submit.

![image](https://github.com/user-attachments/assets/0e1dab4d-f760-4cff-9fcd-5c6a99121305)

**Comment by MarkusBe** [comment]
I have the exact s ame problem. I am using a beelink mini PC for my HA imstande, its bluetooth strength is that high, that i am always switching between the correct room i am currently in and the room where the mini PC is located in.

**Comment by agittins** [comment]
@MarkusBe have you tried the solution?

**Comment by MarkusBe** [comment]
Yes, I just tried it out. Since I am quite new to Homeassistant I first had to figure out how to install a not officially published version of an integration... But I found out, installed it, and tried out. And: it works! Great job... Had to figure out first in which direction I have to adjust the RSSI offset, but now it is working much better than before. I still need some more ESP32 scanners, to make it work in the way I want it too. Thank you very much for this great integration, I love it!

**Comment by agittins** [comment]
Awesome!

Sorry you had to jump through hoops a bit to get the beta version installed, but glad you stuck with it and worked it out!

**Comment by agittins** [comment]
I've decided to close this issue as the **present** use-case of taming very sensitive proxies (or even excluding them from being able to "win" a closest-proxy contest) is sufficiently addressed by adjusting the per-proxy ref_power offset, introduced in versions from v0.6.9rc2 and onwards.

The separate (future) requirement of completely disabling a proxy from contributing to measurements will only apply to trilateration, so won't actually be an issue until that exists, and will likely be treated within that work, so doesn't need to be tracked separately at this time.

TL/DR: This present issue is fixed/worked-around by using v0.6.9rc2 or later, and using the per-proxy calibration.
